### PR TITLE
[TE] Add UI and email template for grouped anomaly

### PR DESCRIPTION
**Pull request by cyenjung**
1. Add a search UI for grouped anomaly. The page maps from a list of group ids to a list of merged anomalies.

2. Add email template for grouped anomaly. The template checks if the alert is a grouped anomaly or a list of merged anomalies. It embeds a link to group anomaly UI for the former case and the original link for the latter case.

Tested on local controller.

### CODE REVIEW

**Comment by kishoreg** [comment]
Do u have screen shots off grouped anomaly.

**Comment by cyenjung** [comment]
Currently, the page for grouped anomaly reuses the page for list of anomaly ids. The only difference is the URL. Later on, we could add user feedback for group anomaly page.

File: thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/HashParams.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ function HashParams() {
   this.ANOMALIES_METRIC_IDS = 'metricIds';
   this.ANOMALIES_DASHBOARD_ID = 'dashboardId';
   this.ANOMALIES_ANOMALY_IDS = 'anomalyIds';
+  this.ANOMALIES_GROUP_IDS = 'groupIds';
```
Comment (kishoreg) [inline thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/HashParams.js]: groupIds is too generic, lets call it anomalyGroupIds

File: thirdeye/thirdeye-pinot/src/main/resources/com/linkedin/thirdeye/detector/multiple-anomalies-email-template.ftl [hunk 2]
```diff
@@ -18,7 +18,11 @@
         <tr>
           <td style="padding: 0 24px;" colspan="2">
               <p style="font-size: 20px; margin-bottom: 8px;">Hi,</p> <br>
-              <p style="color: rgba(0,0,0,0.55); margin-top: 0px;"> ThirdEye has detected <strong style="color: black;">${anomalyCount} ${(anomalyCount == 1)?string("anomaly", "anomalies")}</strong>. Below is a summary, please go <strong><a style="color:#33aada;" href="${dashboardHost}/thirdeye#anomalies?anomaliesSearchMode=id&anomalyIds=${anomalyIds}">here</a></strong> for a detailed view.</p>
+              <#if isGroupedAnomaly>
+                <p style="color: rgba(0,0,0,0.55); margin-top: 0px;"> ThirdEye has detected <strong style="color: black;">${anomalyCount} ${(anomalyCount == 1)?string("anomaly", "anomalies")}</strong>. Below is a summary, please go <strong><a style="color:#33aada;" href="${dashboardHost}/thirdeye#anomalies?anomaliesSearchMode=groupId&groupIds=${groupId}">here</a></strong> for the up-to-dated view of this group of anomalies.</p>
```
Comment (kishoreg) [inline thirdeye/thirdeye-pinot/src/main/resources/com/linkedin/thirdeye/detector/multiple-anomalies-email-template.ftl]: keep the text same as before and change the link. We can revisit this once we have some feedback

**Pull request merged**